sub-packages/bionemo-esm2/src/bionemo/esm2/data/datamodule.py

```diff
@@ -18,19 +18,21 @@
 import os
 from typing import Literal
 
-import pytorch_lightning as pl
-import torch
-import torch.utils.data
+from nemo.lightning.data import WrappedDataLoader
```

Comment:
I wonder if we should do something like this instead.
`import nemo.lightning.data.WrappedDataLoader as MegatronDataLoader`

Reply:
@jstjohn

Reply:
Just to be consistent with `MegatronDataModule`.